gateway/README.md

```diff
@@ -769,7 +769,7 @@ The following is a full list of overrideable options and their defaults:
 
 | Configuration property | Definition | Example values |
 |--------|----------|--------|
-| `StatsDelay` | If and how often we should self report metrics about the gateway out all forwarders | "1s" |
+| `EmitDebugMetrics` | If you want internal metrics reported | false |
 | `ServerName` | All self-reported metrics emitted from the gatway have a `host` dimension. If this is not set it will be the name of current host system | "gateway-staging-1" |
```

Comment:
Is: All self-reported metrics emitted from the gatway 
change to: All self-reported metrics emitted from the gateway